collector/src/lib.rs

```diff
@@ -620,7 +622,14 @@ impl<V: vdaf::Collector> Collector<V> {
             // poll_once() already retries upon server and connection errors, so propagate any error
             // received from it and return immediately.
             let retry_after = match self.poll_once(job).await? {
-                PollResult::CollectionResult(aggregate_result) => return Ok(aggregate_result),
+                PollResult::CollectionResult(aggregate_result) => {
+                    info!(
```

Comment:
IMO, we shouldn't log at `info` for happy-path operations in a library. Drop this to a lower level?